contribs/gmf/src/services/themesservice.js

```diff
@@ -72,6 +76,21 @@ goog.inherits(gmf.Themes, goog.events.EventTarget);
 
 
 /**
+ * Init the service.
+ * @param {string} treeUrl URL to "themes" web service.
+ * @param {string} isThemePrivateUrl URL to check if theme is public.
+ * @export
+ */
+gmf.Themes.prototype.init = function(treeUrl, isThemePrivateUrl) {
```

Comment:
Luxembourg has its own service for the moment, when they want to migrate to use the ngeo one, then we'll see what are the impacts. They probably need to update their application code anyway.